### PR TITLE
Update content for external_scripts/external_stylesheets usage in R

### PR DESCRIPTION
**Pull request by rpkyle**
This PR proposes to update the R docs to include newly available support for setting attributes in `external_scripts` and `external_stylesheets`, as provided by https://github.com/plotly/dashR/pull/226.

### CODE REVIEW

File: dash_docs/chapters/external_resources/examples/addingExternalCSS.R [hunk 1]
```diff
@@ -1,16 +1,25 @@
 library(dash)
 library(dashHtmlComponents)
 
+# external JavaScript files
 external_scripts <- list(
-  list('https://www.google-analytics.com/analytics.js'),
-  list(src = 'https://cdn.polyfill.io/v2/polyfill.min.js'),
-  list(src = 'https://cdnjs.cloudflare.com/ajax/libs/lodash.js/4.17.10/lodash.core.js')
+  src = 'https://www.google-analytics.com/analytics.js',
+  src = 'https://cdn.polyfill.io/v2/polyfill.min.js',
```
Comment (alexcjohnson) [inline dash_docs/chapters/external_resources/examples/addingExternalCSS.R:7]:
```suggestion
  'https://www.google-analytics.com/analytics.js',
  'https://cdn.polyfill.io/v2/polyfill.min.js',
```
Anyway some of these changes conflict with #994 - maybe we just want to update that one to include examples with the full attribute syntax?

Comment (rpkyle) [reply dash_docs/chapters/external_resources/examples/addingExternalCSS.R:7]: Yeah, this branch has been held for weeks due to the `glue` error @HammadTheOne resolved, I apologize as this is a bit inconsistent given recent edits to #994. @alexcjohnson I'm happy to use the abbreviated syntax here too.

**Review by Marc-Andre-Rivet — APPROVED** [review]
💃

File: dash_docs/chapters/external_resources/index.R [hunk 2]
```diff
@@ -179,11 +179,11 @@ examples$embeddingImages$source_code,
...
-You can add resources hosted externally to your Dash app with the 
+You can add resources hosted externally to your Dash app with the
 `external_stylesheets/stylesheets` init keywords.
 
 The resources can be either a string or 
-a dict containing the tag attributes (src, integrity, crossorigin, etc). 
+a list containing the tag attributes (`src`, `integrity`, `crossorigin`, etc).
 You can mix both.
             
 External css/js files are loaded before the assets.
```
Comment (HammadTheOne) [inline dash_docs/chapters/external_resources/index.R:189]: Do we need all this commented code underneath this line or can that be removed?

Comment (rpkyle) [reply dash_docs/chapters/external_resources/index.R:189]: :100: I'll clean that up, thanks!

Comment (rpkyle) [reply dash_docs/chapters/external_resources/index.R:189]: Looks like there's quite a bit of additional content we could add to this page, I'll add it to my list of things to do for next week, so we can bring this to parity with Python insofar as it's possible.

**Pull request merged**